x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -131,9 +131,16 @@ export function ServiceOverview({
             </EuiFlexItem>
             {!isRumAgent && (
               <EuiFlexItem>
-                <EuiPanel>
-                  <ServiceOverviewInstancesTable serviceName={serviceName} />
-                </EuiPanel>
+                <EuiFlexGroup
+                  direction={rowDirection}
+                  gutterSize="s"
+                  responsive={false}
+                >
+                  <ServiceOverviewInstancesRow
```

Comment:
Minor detail but I initially thought this was a single row in the table. I now understand it's a "row" in the service overview grid.

Perhaps renaming it to the slightly verbose `ServiceOverviewInstancesChartAndTable` will make it less ambigious?